src/Avalonia.Input/Platform/IStandardCursorFactory.cs

```diff
@@ -1,9 +1,12 @@
 using Avalonia.Input;
 
+#nullable enable
+
 namespace Avalonia.Platform
 {
     public interface IStandardCursorFactory
```

Comment:
Should this be renamed to `ICursorFactory` or did you intend custom cursors to be created using a separate interface @kekekeks ?

Reply:
Rename